Website/src/rendering/src/stories/Speakers/SpeakersGrid.stories.tsx

```diff
@@ -25,7 +25,7 @@ const speaker1 = {
     },
   },
   featured: {
-    value: true,
+    value: false,
```

Comment:
Please also update the pictures of the speakers for them to be square and transparent backgrounds like the ones we see on the website.

Reply:
This was not done. I am unresolving the conversation